tests/requirements.txt

```diff
@@ -0,0 +1,8 @@
+dimod==0.12.3
```

Comment:
This is meant to be called in addition to the main directory `requirements.txt`. So I would only put the things in here that the tests need that the package doesn't. E.g. `pandas` if we drop that as a direct dependency.

Reply:
Unresolving, I meant this file should be empty.